src/client.js

```diff
@@ -270,25 +271,43 @@ Client.prototype.input = function(data) {
 	var client = this;
 	var text = data.text.trim();
 	var target = client.find(data.target);
+
 	if (text.charAt(0) !== "/") {
-		text = "/say " + text;
+		text = "say " + text;
+	} else {
+		text = text.substr(1);
+
+		// If after removing first / we still have a /, send it as a message
+		if (text.charAt(0) === "/") {
+			text = "say " + text;
+		}
```

Comment:
Definitely less readable though, and suggested solution will definitely not have any performance impact.